sormas-app/app/src/main/java/de/symeda/sormas/app/backend/facility/Facility.java

```diff
@@ -60,6 +62,16 @@
 	private Community community;
 	@Column(length = COLUMN_LENGTH_DEFAULT)
 	private String city;
+	@Column(length = COLUMN_LENGTH_DEFAULT)
+	private String postalCode;
+	@Column(length = COLUMN_LENGTH_BIG)
+	private String street;
+	@Column(length = COLUMN_LENGTH_DEFAULT)
+	private String houseNumber;
+	@Column(length = COLUMN_LENGTH_BIG)
+	private String additionalInformation;
+	@Column
```

Comment:
Don't know why this is written this way in Location.java, but it should actually be `@Enumerated(EnumType.STRING)`